sample/.gitignore

```diff
@@ -5,3 +5,6 @@ vendor/
 resources/*.zip
 resources/*.tgz
 docker-cache/
+
+!resources/no-op-release.tgz
```

Comment:
what does the `!` do?

Reply:
`!` unignores. As in, `*.tgz` is ignored, with the exception of these two specific releases.